codebase/superdarn/src.idl/app/examples.1.2/readcfit.pro

```diff
@@ -1,3 +1,30 @@
+;readcfit.pro
+;==============
+;Author: R.J.Barnes
+
+;Copyright (c) 2012 The Johns Hopkins University/Applied Physics Laboratory
```

Comment:
Maybe this goes against a style guide I am unaware of, but I personally find comments in IDL (and other languages) to be more legible when there is a space between the comment character and text.

Reply:
We don't have a style guide, but I that's fair enough about the spacing.